src/main/java/io/mosip/print/service/impl/PrintServiceImpl.java

```diff
@@ -232,7 +286,7 @@ public boolean generateCard(EventModel eventModel) {
 	private Map<String, byte[]> getDocuments(String credential, String credentialType, String encryptionPin,
 			String requestId,
 			String cardType,
-			boolean isPasswordProtected) {
+			boolean isPasswordProtected, String refId, String registrationId) {
```

Comment:
why is refId passed and not getting used ?

Reply:
removed the unused arguments.